packages/core/src/tracing/cls.js

```diff
@@ -218,6 +218,33 @@ class InstanaPseudoSpan extends InstanaSpan {
     }
   }
 }
+/**
+ * This class represents an ignored span when accessing the `currentSpan`.
```

Comment:
```suggestion
* This class represents an ignored span. We need this representation for the endpoint filtering feature because
 * when a customer accesses the current span via `instana.currentSpan()`, we do not want to return a "NoopSpan".
 * Instead, we return this ignored span instance so the trace ID remains accessible.
```